site/docs/components/scrim/examples.mdx

```diff
@@ -11,39 +11,31 @@ data:
 
 <LivePreviewControls>
 
-<LivePreview componentName="scrim" exampleName="Default">
+<LivePreview componentName="scrim" exampleName="FillViewport">
 
-### Default
+### Fill Viewport
```

Comment:
What's the use case for Scrim to fill the viewport, would we not recommend to use the Dialog component instead ?

Reply:
Scrim is different to Dialog. Dialog is a semantic element that contains a focus trap and other behaviour. Scrim's only purpose is to obscure what's behind it. Dialog comes with a Scrim built in

Reply:
Do we have recommendations  on when to use one vs the other and should this be mentioned in the Usage.mdx ?

Reply:
I don't think it's high priority to add that guidance. We can add it when we work on Dialog this sprint if it's needed.